### PR TITLE
Add 2019 sponsor logos

### PR DESCRIPTION
**Pull request by Bubblemelon**
This PR contains a sponsor section template that can be used/improved for future conferences :smile:. See PR commit history for detailed changes. 

Related issues: 
- [#44](https://github.com/JuliaCon/JuliaCon2019/issues/44). (Contains screenshots of the layout.)
- #189

### CODE REVIEW

File: 2019/sponsors19.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<div class="container-fluid">
+    <div class="container sponsors">
+
+        <!-- Tier levels -->
+        {% assign l1 = "platinum" %}
+        {% assign l2 = "gold" %}
+        {% assign l3 = "sliver" %}
```
Comment (fredrikekre) [inline 2019/sponsors19.html]: `silver`? Here and on multiple more places in this file.

Comment (xorJane) [reply 2019/sponsors19.html]: Good catch!

Comment (Bubblemelon) [reply 2019/sponsors19.html]: :joy: Fixing that !

**Comment by aviks** [comment]
Thanks a ton, @Bubblemelon

**Pull request merged**